File: baseproject-android/app/src/main/java/com/chang/template/activity/BaseView.java
package com.chang.template.activity;

/**
 * Created by Howard Chang on 2016/11/29
 */
public interface BaseView {
    void onRequestStart();

    void onRequestError(String msg);

    void onRequestEnd();

    void onInternetError();

    void onRequestMessage(String msg);
}

File: baseproject-android/app/src/main/java/com/chang/template/activity/BaseView.kt
package com.chang.template.activity

/**
 * Created by Howard Chang on 2016/11/29
 */
interface BaseView{
    fun onRequestStart()

    fun onRequestError(msg: String)

    fun onRequestEnd()

    fun onInternetError()

    fun onRequestMessage(msg: String)
}
